app/views/users/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>specific user</h1>
\ No newline at end of file
+<div class="mb-3">
+  <div class="mb-3">
+    <%= render 'partials/users', user: @user %>
+  </div>
+  <p class="card pb-3 ps-1">
+    <span class="mb-3">Bio</span>
```

Comment:
- [optional] nice job following the wireframe. I understand styling was not the top priority, please kindly make the text a little bigger and bolder, and also fix the margins. This will make it more appealing . Nice job.

Reply:
Hi @codecaiine, @tufoinnkuo10 
Thanks for taking the time to review my project. Your valuable feedback and insights are instrumental in shaping its final outcome. I am truly grateful for your expertise and the effort you put into providing constructive feedback:handshake: